dev_scripts/find_images_to_rebuild.py

```diff
@@ -50,5 +58,15 @@
     dry_run=True, **kwargs)
 
 handler = RebuildImagesOnRPMAdvisoryChange()
-with patch("freshmaker.consumer.get_global_consumer"):
-    handler.handle(event)
+
+if args.cassete_path:
+    import vcr
+    my_vcr = vcr.VCR(
```

Comment:
This code already initializes vcrpy:
https://github.com/redhat-exd-rebuilds/freshmaker/commit/687072cfba2d2474bbf72e4a4359c8c3df794ddf#diff-ca295b25c8157b5a17eb8e3054e8c49dR526-R533

I have not confirmed this, but I think you could create a temporary directory and set `conf.vcrpy_path` to that temporary directory. Then create a symlink to the cassette file in the temporary directory so that it is named after the event created above (e.g. `f'{event.id}.yml'` (verify event.id is actually set)). Then if you set `conf.record_mode` to `"none"`, it should automatically pick up the cassette and use it.